playbooks/init/main.yml

```diff
@@ -12,28 +12,16 @@
       aggregate: false
 
 - include: evaluate_groups.yml
```

Comment:
Do we want to rename these files as well?  Since we are in `init` it makes sense that we could rename this to `facts.yml`.  Same as the others in this file.

I'd also suggest removing `openshift` from the file names as well.

we'd have
```
facts.yml
sanity_checks.yml
validate_hostnames.yml
repos.yml
version.yml
```

Thoughts?

Reply:
I don't mind making the filename changes.